test/test_integration.py

```diff
@@ -496,3 +496,27 @@ def update_graph(n_clicks):
         button.click()
         time.sleep(2)
         self.snapshot('candlestick - 2 click')
+
+    def test_latex(self):
+        app = dash.Dash(__name__)
+        app.layout = html.Div([
+
+            html.Label('Graph'),
+            dcc.Graph(
+                id='graph',
+                figure={
+                    'data': [{
+                        'x': [1, 2, 3, 4, 5],
+                        'y': [1, 3, 5, 9, 13]
+                    }],
+                    'layout': {
+                        'title': 'sqrt{(n_\\text{c}(t|{T_\\text{early}}))}$'
```

Comment:
we're missing a leading `$` here, I believe it should be 
```
$sqrt{(n_\\text{c}(t|{T_\\text{early}}))}$
```

Reply:
I must've missed that while testing with different inputs. Fixed this. Check perci.io output now, hopefully, this should work.